web/client/translations/data.it-IT.json

```
@@ -696,6 +696,7 @@
             "newMeasure": "Nuova annotazione",
             "selectTool": "Seleziona uno strumento di misura",
             "exportToGeoJSON": "Esporta in GeoJSON",
+            "saveMeasure": "Salva la misurazione nell'annotazione",
```

i think is better to use plural by default, please review also other translations

```suggestion
            "saveMeasure": "Salva le misure nell'annotazione",
```